### PR TITLE
feat(video): add support for video ad volume control

### LINKED ISSUE #580: [🐛] MobileAds.setAppVolume and MobileAds.setAppMuted
### What happened?

Hi,

Is it possible to use MobileAds.setAppVolume and MobileAds.setAppMuted ?

Thank you in advance

### Platforms

Android and iOS

### React Native Info

```Shell
Not important
```

### Are your using Typescript?

- [ ] My project is using Typescript

### package.json

```JSON
Not important
```

### app.json

```JSON
Not important
```

### ios/Podfile

_No response_

### android/build.gradle

_No response_

### android/app/build.gradle

_No response_

### android/settings.gradle

_No response_

### AndroidManifest.xml

_No response_

### PR DESCRIPTION
**Pull request by adrienlfr**
### Description

I need to be able to control the volume of ads in my application.

Here's the documentation for volume control: [Video ad volume control](https://developers.google.com/ad-manager/mobile-ads-sdk/ios/global-settings?hl-fr#video_ad_volume_control)

### Related issues

Fixes #580 

### Release Summary

Implement Video ad volume control

### Checklist

- I read the [Contributor Guide](https://github.com/invertase/react-native-google-mobile-ads/blob/main/CONTRIBUTING.md)
  and followed the process outlined there for submitting PRs.
  - [ ] Yes
- My change supports the following platforms;
  - [x] `Android`
  - [x] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `__tests__e2e__`
  - [x] `jest` tests added or updated in `__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

image

---

Think `react-native-google-mobile-ads` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

File: __tests__/googleMobileAds.test.ts [hunk 1]
```diff
@@ -87,6 +87,22 @@ describe('Admob', function () {
...
+
+      it('does call native setAppVolume method', () => {
+        admob().setAppVolume(0.5);
+        expect(RNGoogleMobileAdsModule.setAppVolume).toBeCalledTimes(1);
+      });
+
+      it('throws if setAppVolume is greater then 1', function () {
+        expect(() => {
+          admob().setAppVolume(2);
+        }).toThrowError('The app volume must be a value between 0 and 1 inclusice.');
```
Comment (dylancom) [inline __tests__/googleMobileAds.test.ts:99]: typo inclusice

**Comment by dylancom** [comment]
Nice addition!

File: __tests__/googleMobileAds.test.ts [hunk 2]
```diff
@@ -87,6 +87,22 @@ describe('Admob', function () {
           admob().openDebugMenu('');
         }).toThrowError('openDebugMenu expected a non-empty string value');
       });
+
+      it('does call native setAppVolume method', () => {
+        admob().setAppVolume(0.5);
+        expect(RNGoogleMobileAdsModule.setAppVolume).toBeCalledTimes(1);
+      });
+
+      it('throws if setAppVolume is greater then 1', function () {
```
Comment (dylancom) [inline __tests__/googleMobileAds.test.ts:96]: greater than 1

File: docs/displaying-ads.mdx [hunk 3]
```diff
@@ -484,3 +484,38 @@ function App() {
 ```
 
 The `sizes` prop takes an array of [`BannerAdSize`](/reference/admob/banneradsize) types.
+
+### Video ad volume control
```
Comment (dylancom) [inline docs/displaying-ads.mdx:488]: I think we should place this under 'Advanced Usage'

**Comment by adrienlfr** [comment]
Thanks for the review. I've corrected the spelling mistakes and updated the doc.

**Comment by dylancom** [comment]
I meant under the "Section" Advanced Usage.
See docs.json:
```
"Advanced Usage",
      [
        ["Displaying Ads via Hook", "/displaying-ads-hook"],
        ["European User Consent", "/european-user-consent"],
        ["Ad Inspector", "/ad-inspector"],
        ["Impression-level ad revenue", "/impression-level-ad-revenue"]
      ]
```

**Comment by dylancom** [comment]
Awesome, thanks for making this library better :)

**Pull request merged**